test/testdata/configs/config-v28.json

```diff
@@ -9,7 +9,7 @@
     "Archival": false,
     "BaseLoggerDebugLevel": 4,
     "BlockServiceCustomFallbackEndpoints": "",
-    "BlockServiceHTTPMemCap": 500000000,
+    "BlockServiceMemCap": 500000000,
```

Comment:
What effect does this have for customers who currently have this value set? Since the name of the value is changing, they won't realize their custom value is no longer being used, right?

Reply:
The original config value hasn't been released yet so it's safe to change. config version 28 is for the upcoming release and this value is introduced in version 28